daemon/top_windows.go

```diff
@@ -26,7 +26,7 @@ import (
 func (daemon *Daemon) ContainerTop(name string, psArgs string) (*types.ContainerProcessList, error) {
 	// It's not at all an equivalent to linux 'ps' on Windows
 	if psArgs != "" {
-		return nil, errors.New("Windows does not support arguments to top")
+		return nil, errors.New("windows does not support arguments to top")
```

Comment:
I would say that Windows should remain capitalized, as it is a proper noun. This could be reworded if we want the error string to start with a lowercase